packages/Manager/src/main/kotlin/nz/govt/eop/rainfall/RainfallQueries.kt

```diff
@@ -56,6 +57,9 @@ class RainfallQueries(
                         "totals.OBSERVATION_MEASUREMENT_ID",
                         OBSERVATION_SITES_MEASUREMENTS.ID.dataType)))
             .where(OBSERVATION_SITES_MEASUREMENTS.MEASUREMENT_NAME.eq(RAINFALL_MEASUREMENT_NAME))
+            .and(
+                OBSERVATION_SITES_MEASUREMENTS.LAST_OBSERVATION_AT.ge(
+                    OffsetDateTime.parse("2024-01-01T00:00:00Z")))
```

Comment:
I gather this hard code is as intended, all observations since they started (start of the year)?

Reply:
Yip, because otherwise we were returning sites where we had some rainfall data from the 1800's and saying there is no data available. I guess in future there should be a more explicit way of saying a site is active